### PR TITLE
WIP: Add support for Cassandra UDT

### LINKED ISSUE #6047: Support for Cassandra UDT
When I query cassandra cf that with UDF, the presto query cli will complain: Query ***\* Failed, CassandraType is null, due to there is support for Cassandra UDF yet: https://github.com/prestodb/presto/blob/master/presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraType.java

### PR DESCRIPTION
**Pull request by corrius**
Add support for Cassandra UDT, serializing the user defined types to json strings.

I have put together a quick test while developing just to make sure that the fix works but I will be polishing that test.

Closes #6047

### CODE REVIEW

**Comment by cawallin** [comment]
@corrius is this still WIP?

File: presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraType.java [hunk 1]
```diff
@@ -248,6 +254,11 @@ private static String buildListValue(Row row, int i, CassandraType elemType)
         return buildArrayValue(row.getList(i, elemType.javaType), elemType);
     }
 
+    private static String buildUDTValue(Row row, int i)
```
Comment (cawallin) [inline presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraType.java]: UDT should be camel case, as it's easier to read (and the rest of the Presto code is formatted like that): e.g. buildUdtValue

**Comment by corrius** [comment]
@cawallin I got distracted into modifying the connector to parse the collections to their 'native' type and not serialize them as String and never finish this one. I will finish the test, make any change based on your review and update the PR.

**Comment by cawallin** [comment]
No rush, I just wanted to know if it was waiting to be reviewed.

**Pull request closed without merging**